### DISCUSSION TITLE
Can have colored background over the entered text like Instagram

### DISCUSSION THREAD

**Discussion by nayanbabariya95** [mehdizarepour/text_editor#13] [Ideas]
I would like to get a feature of various colored background over the entered text as in the Instagram app.

**Comment by mehdizarepour** [comment]
Hi, @nayanbabariya95.

Can you share more details or send a picture?

**Comment by nayanbabariya95** [comment]
![Screenshot_20211116-212251_Instagram](https://user-images.githubusercontent.com/63590009/142019397-9321b566-cb25-48b2-a8b9-cff2aa551bf6.png)

This is the effect (White background on only entered text)

**Reply by mehdizarepour** [reply]
We already have the background feature, have you checked the latest version?
